CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+#### Changes
+* The `:ignore-errors` option will be honored in more places, making refactor-nrepl more robust in face of files not particularly meant to be part of the AST corpus.
```

Comment:
I'd also document this more prominently somewhere, as there are a couple of other editors using `refactor-nrepl`.